CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## Main (unreleased)
 
-- Exclude development-only files from the packaged buildpack
```

Comment:
Was this entry meant to be removed?

Reply:
Yes, I mentioned in the commit message why. It's not a customer facing change (or shouldn't be anyway).